target/linux/rockchip/image/armv8.mk

```diff
@@ -4,6 +4,27 @@
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
+
+define Device/friendlyelec_nanopi-r2-rev00
+  DEVICE_VENDOR := FriendlyELEC
+  DEVICE_MODEL := NanoPi R2S
```

Comment:
So, the device should be named friendlyelec_nanopi-r2s ...?
Why has the name of the device node been chosen differently?